### PR TITLE
overlay: freeze libsolv

### PR DESCRIPTION
**Pull request by ignatenkobrain**
Due to changelog format change.

### CODE REVIEW

File: overlays/dnf-master/overlay.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ components:
   - name: libsolv
     git:
       src: github:openSUSE/libsolv.git
+      freeze: 335f469bf6e3109d4973e55a3a86a274adb0c21d
```
Comment (m-blaha) [inline overlays/dnf-master/overlay.yml]: this freeze refers to distgit commit, not upstream

Comment (ignatenkobrain) [reply overlays/dnf-master/overlay.yml]: whoops, wrong section. Can you fix it?

Comment (m-blaha) [reply overlays/dnf-master/overlay.yml]: Yes, I'll do it

**Comment by m-blaha** [comment]
Fixed by https://github.com/rpm-software-management/ci-dnf-stack/commit/9b26ca52d197f9121e5bf01c2bce35baa893b3c1

**Pull request closed without merging**